### DISCUSSION TITLE
BaseURL of zodios client

### DISCUSSION THREAD

**Discussion by TheSumm** [kubb-labs/kubb#630] [Q&A] [answered]
I am using `@kubb/swagger-zodios` to generate a zodios client. The generated api in zodios.ts does not use the Zodios constructor with a baseURL: `export const api = new Zodios(endpoints);`. So I am wondering what is the intended way to configure the client with a baseURL?

Config:
```js
import { defineConfig } from "@kubb/core";
import createSwagger from "@kubb/swagger";
import createSwaggerZod from "@kubb/swagger-zod";
import createSwaggerZodios from "@kubb/swagger-zodios";

export default defineConfig({
  root: ".",
  input: {
    path: "./apis/swagger.json",
  },
  output: {
    path: "./src/gen",
    clean: true,
  },
  plugins: [
    createSwagger({ output: false }),
    createSwaggerZod({
      output: "./zod",
    }),
    createSwaggerZodios({
      output: "./zodios",
    }),
  ],
});
```

**Comment by stijnvanhulle** [comment, marked as answer]
The idea is to use `servers[index].url` specified in your Swagger/OpenAPI file(see [example](https://github.com/kubb-project/kubb/blob/426a3ff33fb94bd11e95953e842341e9facad88e/examples/advanced/petStore.yaml#L24)).

With the `swagger plugin` you can choose which index to use: https://www.kubb.dev/plugins/swagger#serverindex and then the `Zodios plugin` will use that as your baseURL. For now this was not yet implemented for Zodios but I opened a PR to add this(https://github.com/kubb-project/kubb/pull/631)
